frontend/src/components/Application/Posts/views/PostEditor.vue

```diff
@@ -2,23 +2,71 @@
   <div class="container mt-2">
     <h1 class="mb-3">Post Something</h1>
     <v-form @submit.prevent="onSubmit">
-      <v-textarea
-        v-model="post_body"
-        class="form-control"
-        placehodler="What do you want to post?"
-        rows="3"
-      ></v-textarea>
+      <v-card>
+        <v-card-text>
+          <v-layout row wrap justify-space-around>
+            <v-flex xs8 sm9 text-xs-center>
+              <v-text-field
+                v-model="post_body"
+                class="form-control"
+                placeholder="What do you want to post?"
+                rows="3"
+              ></v-text-field>
+              <p v-if="error" class="grey--text">{{ error }}</p>
+              <p v-else class="mb-0">
+                <v-text-field v-model="speech" v-if="sentences.length > 0" v-bind:key="sentences" 
+                  :value="sentences">
+                </v-text-field>
+                <span v-for="sentence in sentences" v-bind:key="sentence">{{sentence}}. </span>
```

Comment:
This might be where those extra periods are coming from. Removing the "." after the sentence string interpolation {{ sentence }} may possible correct this, though the original author may have wanted this behavior.

Reply:
Thanks! I fixed that.

Reply:
Additionally, this span is the Vue equivalent of a for-loop, rendering every sentence stored in an array and printing it here. The 'v-for' directive in Vue can be used with other HTML elements, like divs or paragraphs, and Vuetify components as well.

Reply:
I've had issues with it not working properly if it's not in a span but feel free to try and change that.